app/app/Http/Controllers/LocationReportController.php

```diff
@@ -35,7 +35,8 @@ public function show(string $location_id, $rating_system = null)
 			$rating_system = 'universal';
 		}
 		$location = Location::find($location_id);
-		$question_categories = QuestionCategory::with('questions')->get();
+		//changes order by
```

Comment:
Does this sort the categories also on the location rating feature and the profile?  Please test that.

Reply:
This sort the categories on location report and location rating feature only. However, I am looking to make this change on profile page too.